daemons/gptp/common/ieee1588port.cpp

```diff
@@ -230,8 +230,11 @@ bool IEEE1588Port::init_port(int delay[4])
 void IEEE1588Port::startPDelay() {
 	if (automotive_profile) {
 		if (log_min_mean_pdelay_req_interval != PTPMessageSignalling::sigMsgInterval_NoSend) {
+			long long unsigned int waitTime;
+			waitTime = ((long long) (pow((double)2, log_min_mean_pdelay_req_interval) *  1000000000.0));
```

Comment:
how about using this instead of pow:

waitTime = log_min_pdelay_req_interval<0 ? 1000000000LL>>(-log_min_pdelay_req_interval) : 1000000000LL<<log_min_pdelay_req_interval;